### PR TITLE
Ra staging 20190522

### PR DESCRIPTION
**Pull request by rhiananthony**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Release Notes
 
+# Staging 2019/05/22 
+## Ingest
+### Exporter v0.8.1.rc
+* Do not inform user when there’s a failure creating a duplicate bundle
+
```
Comment (aaclan-ebi) [inline RELEASE_NOTES.md]: @ranthony sorry, I just added some more release notes for Ingest UI that was also deployed to staging today. Could you add it here?

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Sure. Np :)

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Done!

Comment (aaclan-ebi) [reply RELEASE_NOTES.md]: Thank you! :)

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+### UI v0.9.4.rc
+* Use Fusillade integration
+
+## Azul
+### Version: bb092dafe26477887baba63d6ed016b8ee2ec4c1
+* 589a787 (gitlab.dev/develop, develop) Fix: BDBag archive contains non-deterministic directory name (#991)
```
Comment (achave11) [inline RELEASE_NOTES.md]:
```suggestion
* 589a787 Fix: BDBag archive contains non-deterministic directory name (#991)
```

I made a last update to the README which is reflected here. Apologies for the mistake.

Comment (rhiananthony) [reply RELEASE_NOTES.md]: Done :)

**Pull request merged**